.github/workflows/integration-test-multi-cluster-suite.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
 
       - name: Artifact
         uses: actions/upload-artifact@4cec3d8aa04e39d1a68397de0c4cd6fb9dce8ec1 # v4.6.1
-        if: failure()
+        if: always()
```

Comment:
was this just for debugging, and can be reverted?

Reply:
I think we can have it as we collect the logs and upload artifacts always for all other tests